docs/source/cloud-setup/cloud-permissions/aws.rst

```diff
@@ -4,36 +4,330 @@
+- **SSO start URL**: copy from the SSO login portal
+- **SSO region**: copy from the SSO login portal
+- **SSO registration scopes**: leave blank to use the default
+
+Log in and approve the request in your web browser. Then back in the CLI, complete the remaining fields:
+
+- **Default client Region**: optional
+- **CLI default output format**: optional
+- **Profile name**: set to ``default`` if you want to use this profile by default with SkyPilot, otherwise see :ref:`several-aws-profiles`.
+
+If everything is set up correctly, :code:`sky check aws` should succeed!
+
+
+.. _dedicated-aws-user:
+
+Setting up a dedicated SkyPilot IAM user
+----------------------------------------
+
+You can create a dedicated IAM user for SkyPilot, if you can't or don't want to use your normal credentials, or if you want to restrict SkyPilot's permissions.
+
+.. note::
+
+  **SSO users** that login via awsapps.com should set up a dedicated user to ensure cross-cloud access. Otherwise, instances launched in other clouds will not be able to access AWS resources, including buckets and managed jobs.
+
+Follow these steps to create a new AWS user:
+
+1. Open the `IAM dashboard <https://us-east-1.console.aws.amazon.com/iamv2/home#/home>`_ in the AWS console and click on the **Users** tab. Then, click **Create User** and enter the user's name. Click **Next**.
+
+   **TODO: fix screenshots**
+
+   .. image:: ../../images/screenshots/aws/aws-add-user.png
+       :width: 80%
+       :align: center
+       :alt: AWS Add User
+
+2. In the **Permissions options** section, select "Attach existing policies directly". Depending on whether you want simplified or minimal permissions, follow the instructions below:
+
+   .. tab-set::
+
+       .. tab-item:: Simplified permissions
+
+           Search for the **AdministratorAccess** policy, and check the box to add it. Click **Next** to proceed.
+
+       .. tab-item:: Minimal permissions
+
+           Click on **Create Policy**.
+
+           .. image:: ../../images/screenshots/aws/aws-create-policy.png
+               :width: 80%
+               :align: center
+               :alt: AWS Create Policy
+
+           This will open a new window to define the minimal policy.
+
+           Choose "JSON" tab and copy the needed minimal policy rules.
+           **See** :ref:`aws-minimal-policy` **for all the policy rules.**
+
+           Click **Next: Tags** and follow the instructions to finish creating the policy. You can give the policy a descriptive name, such as ``minimal-skypilot-policy``.
+
+           Go back to the previous window and click on the refresh button, and you can now search for the policy you just created.
+
+           .. image:: ../../images/screenshots/aws/aws-add-policy.png
+               :width: 80%
+               :align: center
+               :alt: AWS Add Policy
+
+           Check the box to add the policy, and click **Next** to proceed.
```

Comment:
Can we move these steps into the `Create a user account with minimal permissions` below and just keep a link here to make it easier to read? Otherwise, it feels to long to scroll down.

Reply:
I see what you mean.

> move these steps into the `Create a user account with minimal permissions`

That section is reduced significantly because most of the flow is the same for minimal permissions and for simple permissions. It doesn't make sense to move everything into the "Minimal permissions" section because these only apply to user creation, and "Minimal permissions" could apply to both user creation and role creation.

That said, I think this can be better. Will do another pass